docker/dev-environment-quick-start-guide.md

```diff
@@ -60,6 +60,11 @@ Optionally — recommended —, also install a test product base with product pi
 $ docker-compose -f docker-compose.yml -f docker-compose.dev.yml exec backend bash /opt/scripts/import_sample_data.sh
 ```
 
+If you are running Docker on Windows, you should edit your hosts file (C:\Windows\System32\drivers\etc\hosts) and add the following line :
+```text
+127.0.0.1 world.productopener.localhost fr.productopener.localhost static.productopener.localhost ssl-api.productopener.localhost fr-en.productopener.localhost 
```

Comment:
For some reason, I didn't have to add this, and *. localhost resolves correctly. What about you, @CharlesNepote and @VaiToN?

Reply:
I had to. I remember having a conversation on Slack with @stephanegigandet I think about this.